test/processCommand_test.js

```diff
@@ -0,0 +1,140 @@
+goog.require('helper');
+
+describe('The processCommand function', function() {
+  /**
+   * A helper method to assert that a command passed to the processCommand
+   * method produces the desired result.
+   * @param {!Array<Object>} command A list of arguments to be given to the
+   * processCommand_function of helper, representing the command to run on the
```

Comment:
Please indent when a parameter annotation comment spans multiple lines: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments